src/docs/options.md

```diff
@@ -285,30 +316,6 @@ When in doubt, the scope is described in the heading for each option in the help
         Compile with debug symbol enabled.
     ...
 
-### Check the Formatting
+### Validate the TOML
 
-Settings that span multiple lines should be indented.  To minimize problems, follow these
-conventions:
-
-  - Followon lines should be indented four spaces.
-  - The ending bracket for lists and dicts should be indented two spaces.
-
-Here are some examples of correctly and incorrectly formatted values:
-
-    :::ini
-    # Right
-    jvm_options: [ "foo", "bar" ]
-
-    # Right
-    jvm_options: [
-        "foo", "bar"
-      ]
-
-    # Wrong
-    jvm_options: [ "foo",
-    "bar" ]  # Followon line must be indented
-
-    # Wrong
-    jvm_options: [
-        "foo", "bar"
-    ] # closing bracket must be indented
+Most editors will warn when you have invalid TOML. Another useful resource is the website [www.toml-lint.com](https://www.toml-lint.com).
```

Comment:
Hrm. Clearly pants.toml parses, but that site can't handle it today. It does validate the other 3 fine though.

Reply:
Oh, huh, that's a bummer. `my_list_option.add` is perfectly valid TOML - it's shorthand for this

```toml
[my_list_option]
add = ["a"]
```

Will remove this from the recommendation..Thanks.